src/coreclr/src/pal/inc/unixasmmacrosarm64.inc

```diff
@@ -14,15 +14,6 @@
 
 .macro NESTED_END Name, Section
         LEAF_END \Name, \Section
-#if defined(__APPLE__)
-        .set LOCAL_LABEL(\Name\()_Size), . - C_FUNC(\Name)
-        .section __LD,__compact_unwind,regular,debug
-        .quad C_FUNC(\Name)
-        .long LOCAL_LABEL(\Name\()_Size)
-        .long 0x04000000 // DWARF
-        .quad 0
-        .quad 0
-#endif
```

Comment:
This code wasn't quite right.  The value for arm64 dwarf is 0x03000000.  However even with that fixed the linker was reporting compact unwind errors.

The comments in the Apple compact unwind header suggests the linker will no longer attempt to create compact unwind entries from dwarf tables.  Specifically because they found it was too fragile.

I chose to remove this so that the linker was forced to use the dwarf unwind info for these hand coded assembly files.

w/o this change the linker reportes errors related to overlapping __compact_unwind entries as well as corrupt size.

IMHO this seems reasonable....